src/commands/commerce/store/create.ts

```diff
@@ -446,9 +446,7 @@ export class StoreCreate extends SfdxCommand {
 
     private async createSearchIndex(): Promise<void> {
         if ((await this.statusFileManager.getValue('indexCreated')) === 'true') return;
-        this.ux.log(
-            msgs.getMessage('create.createSearchIndexInfo', ['https://github.com/forcedotcom/sfdx-1commerce-plugin'])
-        );
+        this.ux.log(msgs.getMessage('create.createSearchIndexInfo'));
```

Comment:
Update messaging because 1commerce-plugin is no longer required to build search indexes